Ryoung27/redditClone
# Ry day three reddit clone

steps to test
pull down code

git fetch --all
git checkout ryDayThreeRedditClone
make migrations
python manage.py makemigrations
python manage.py migrate
start server with python manage.py runserver
Check browsable API and make sure a comment was created.
